test/integration/openid4vp_test.go

```diff
@@ -201,4 +212,16 @@ func TestOpenID4VPFullFlow(t *testing.T) {
 
 		fmt.Printf("done test %d\n", i)
 	}
+
+	time.Sleep(5 * time.Second)
```

Comment:
Reason for the `time.Sleep`? These will slow down our tests

Reply:
logs will not appear on the traces collector immediately, some timeout needed to wait until it will process them

Reply:
OK